### PR TITLE
Add automated issue creation for PHP Core Roundup

### PR DESCRIPTION
**Pull request by pronskiy**
@Ayesh, @saundefined, I want to add an automatic issue as a reminder and a checklist of all things to do for roundup.

### CODE REVIEW

File: .github/workflows/twitter.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: twitter-together/action@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  tweet:
+    name: Tweet
+    runs-on: ubuntu-latest
+    environment: is-php-dying
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```
Comment (saundefined) [inline .github/workflows/twitter.yml:16]: Haven't had time to read `twitter-together` in detail yet, so just for clarification:

If we create a preview of a tweet using `Preview` job, and after that there is any other non-PR related push, the tweet will not be published?

Anyway PR looks good, thanks a lot!

Comment (pronskiy) [reply .github/workflows/twitter.yml:16]: As I understand, the logic of getting new tweets is independent between `pull_request` and `push` events so it should work:
https://github.com/twitter-together/action/blob/main/lib/push/get-new-tweets.js
https://github.com/twitter-together/action/blob/main/lib/pull-request/get-new-tweets.js

**Comment by pronskiy** [comment]
Regarding Twitter, I was thinking whether it should be in a separate repo though or this one is good?

**Comment by pronskiy** [comment]
As @saundefined mentioned, the main concern is access rights. Let's try to use twitter-together in this repo, and if we see any issues, I'll move it to a separate one.

**Pull request merged**